webapp/login/views.py

```diff
@@ -70,6 +71,9 @@ def after_login(resp):
             "image": resp.image,
             "email": account["email"],
         }
+        owned, shared = logic.get_snap_names_by_ownership(account)
+        flask.session["user_shared_snaps"] = list(shared)
```

Comment:
Do we need to call `list` on shared? Seems like the `logic.get_snap_names_by_ownership` function already returns a list.

Reply:
Ah, good catch, it's a leftover from old code then keys needed converting to list. Will fix that.